spec/system/batch_process_spec.rb

```diff
@@ -29,15 +29,6 @@
       expect(BatchProcess.count).to eq 1
       expect(page).to have_content("Your records have been retrieved from the MetadataCloud. PTIFF generation, manifest generation and indexing happen in the background.")
       expect(BatchProcess.last.file_name).to eq "short_fixture_ids.csv"
-      within "td.process_id" do
-        expect(page).to have_link(BatchProcess.last.id.to_s, href: "/batch_processes/#{BatchProcess.last.id}")
-      end
-      within "td.count" do
-        expect(page).to have_content('5')
-      end
-      within "td.view" do
-        expect(page).to have_link('View', href: "/batch_processes/#{BatchProcess.last.id}")
-      end
```

Comment:
Removed because the td classes no longer exist now that we're using a data table and not a bootstrap table. these values are being checked in `spec/datatables/batch_process_datatable_spec.rb`